framework/src/modules/nft/endpoint.ts

```diff
@@ -153,30 +153,42 @@ export class NFTEndpoint extends BaseEndpoint {
 			return { collectionIDs: [] };
 		}
 
+		const isNFTSupported = await this.isNFTSupported(context);
```

Comment:
Similar to the `isCollectionIDSupported`, the schema for the request parameter verified against `getSupportedCollectionIDsRequestSchema` contains only `chainID` only and `isNFTSupported` requires `id` field, which isn't defined in the exposed schema for the endpoint. I guess the call to `isNFTSupported` is not required.

Reply:
Same ans as above